### PR TITLE
Feature/kak/travel distances not times#1009

### LINKED ISSUE #1009: Use absolute distance instead of travel time
Instead of calculating travel time to all locations in the list, just show their absolute distance from the origin, if an origin is set.

### PR DESCRIPTION
**Pull request by flibbertigibbet**
## Overview

Instead of finding the travel time to each destination and displaying it, show simple distance instead.

Add `turf.js` distance library to support calculating distances client-side, to take advantage of the destinations cache.

Also slightly modified line height of the distance field on the place card, to fix the bottom of some letters getting cut off.

### Demo

distances_homepage

Short distance formatting:
formatted_short_distance

Exactly one mile formats as singular:
formatting_one_mile_exactly

## Testing Instructions

 * Place cards should display no distance when origin not set
 * On setting origin, distances should display on place cards
 * Dragging to reroute should still work (check existing turf function)

Closes #1009

### CODE REVIEW

File: src/app/scripts/cac/places/cac-places.js [hunk 1]
```diff
@@ -79,77 +79,57 @@ CAC.Places.Places = (function(_, $, moment, Routing, UserPreferences) {
...
+        var originLabel;
+
+        if (origin) {
+            originLabel = origin.name ? origin.name :
+                (origin.attributes && origin.attributes.StAddr ? origin.attributes.StAddr :
+                UserPreferences.getPreference('originText'));
+        }
+
+        if (exploreLatLng) {
+            var from = turf.point(exploreLatLng);
```
Comment (KlaasH) [inline src/app/scripts/cac/places/cac-places.js]: Looks like [turf-point wants `(lon, lat)` pairs](https://www.npmjs.com/package/turf-point), not the other way around.  This explains why it's saying my house is only 1.1 miles from Fairmount Waterworks--at -75° latitude, a .08° longitude difference is not so far.

**Comment by flibbertigibbet** [comment]
build again

**Comment by flibbertigibbet** [comment]
build again

**Comment by tnation14** [comment]
@azavea-bot rebuild

**Comment by flibbertigibbet** [comment]
build again

File: src/app/scripts/cac/places/cac-places.js [hunk 2]
```diff
@@ -79,77 +79,57 @@ CAC.Places.Places = (function(_, $, moment, Routing, UserPreferences) {
...
-                    }
-                }).fail(function(error) {
-                    console.error('error finding travel time to ' + destination.name);
-                    console.error(error);
-                    dfd.reject(error);
-                });
-            } else {
-                // event without a location
-                dfd.resolve(destination);
+                var to = turf.point([yCoord, xCoord]);
```
Comment (KlaasH) [inline src/app/scripts/cac/places/cac-places.js]: Should be `[xCoord, yCoord]`.

Comment (flibbertigibbet) [reply src/app/scripts/cac/places/cac-places.js]: Yes. Was waiting for a good CI build before revisiting.

**Comment by flibbertigibbet** [comment]
Updated for coordinate swap.

File: src/app/scripts/cac/places/cac-places.js (hunk 2 above)
Comment (KlaasH) [reply src/app/scripts/cac/places/cac-places.js]: Oops, I forgot to reassign before, otherwise I would have known that :)

**Pull request merged**